web/src/app-main.ts

```diff
@@ -100,15 +102,34 @@ export class AppMain extends LitElement {
         </a>
       </footer>
 
-      <sl-drawer label="About Nixpkgs Graph Explorer" id="drawer-info">
-        A
-        <a href="https://tweag.io">Tweag</a>'s project.
-        <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</div>
+      <sl-drawer label="About Nixpkgs Graph Explorer" id="drawer-info" style="display: flex; flex-direction: column;">
```

Comment:
We have the styles in the `css` section, we can move the styles to line 211, to the `#drawer-info` selector